src/vs/workbench/browser/parts/editor/multiEditorTabsControl.ts

```diff
@@ -1516,6 +1541,9 @@ export class MultiEditorTabsControl extends EditorTabsControl {
 	}
 
 	getHeight(): number {
+		if (!this.visible) {
+			return 0;
+		}
 
```

Comment:
I think this code is not needed given `computeHeight` takes care?

Reply:
Sure I will do it, however, I will also have to invalidate the cacheing of this.dimensions.used when hiding the tab bar, which actually does sort of make sense